quantum/process_keycode/process_music.c

```diff
@@ -8,36 +17,41 @@ int music_offset = 7;
 static bool music_sequence_recording = false;
 static bool music_sequence_recorded = false;
 static bool music_sequence_playing = false;
-static float music_sequence[16] = {0};
+static uint8_t music_sequence[16] = {0};
```

Comment:
Yeah, the one advantage I could see to using a float here is you could theoretically record things like pitch bends that are between notes in the chromatic scale, but I think once you go down that road the logical conclusion is recording full samples which will use a lot more memory.